packages/flutter/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 * update example app to use Android embedding v2 ([#722](https://github.com/parse-community/Parse-SDK-Flutter/issues/722)) ([e092189](https://github.com/parse-community/Parse-SDK-Flutter/commit/e092189cb666c25b3e2c9dbbf95316e9cfa88e72))
 
+### Refactoring
```

Comment:
Could you move this to the top under a new version 3.1.2?
And rename the title to "Refactor".